### PR TITLE
Sprint 7

### PR DESCRIPTION
**Pull request by GregoryForrest**
Change Contractor Name limit to 100 characters
Uppercase the first letter of names, and lowercase remaining
Improve how phone input/formatting works
Remove RPA file not needed by CAFTOP
Remove comments and console log
Add new sections to Homepage - Relocate "Create" button
Update number of items on dashboards
Fix TechnicalOrders not being sent to the document
Update so complete validation isn't run until they are on the final step -- improving performance and code-splitting

Document Updates:
Add commas to cost values
Add section 8.0 Technical Order Manager Approval
Check for Paper/CD/DVD counts in addition to Distribution to ensure "bad" data when changing between electronic only and not
Modify navigation to prompt for save if data changed

Lint fixes:
Fast refresh for all Uppercase named components
Assert type on JSON.parse
Better typing paged iterator
Provide missing props to useEffect dependecy array
Returned promises
Split GlobalStore into 2 files to support fast refresh

### CODE REVIEW

File: src/utilities/Names.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const capitalizeFirstLetter = (str: string) => {
+  if (str === "") {
+    return "";
+  }
+
+  return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
```
Comment (RobertPorterfield) [inline src/utilities/Names.ts:6]: I'm always surprised this isn't a standard function.

File: src/utilities/Phone.ts [hunk 2]
```diff
@@ -1,30 +1,17 @@
...
-  let retVal = phone;
-  if (phone.match(/-$/)) {
-    endsDash = true;
+  if (phone.match(/^([^\d]*\d){10}$/)) {
+    // Match only if it has exactly 10 digits
+    const justNumbers = phone.replace(/\D/g, ""); // Strip out non-digits
+    return `(${justNumbers.slice(0, 3)}) ${justNumbers.slice(
+      3,
+      6
+    )}-${justNumbers.slice(6)}`; // Return number formated as (###) ###-####
```
Comment (RobertPorterfield) [inline src/utilities/Phone.ts]: Should there be a fallback if it has 7 digits?

Comment (GregoryForrest) [reply src/utilities/Phone.ts]: Added one

File: src/api/CAFTOP/useFPCAFTOPs.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ type pageType = {
   hasMore: boolean;
 };
 
-export const PAGESIZE = 3;
+export const PAGESIZE = 15;
```
Comment (RobertPorterfield) [inline src/api/CAFTOP/useFPCAFTOPs.ts:19]: Good change! 🤣

File: src/Steps/Complete.tsx
Comment (RobertPorterfield) [inline src/Steps/Complete.tsx:1]: Not part of the changes for this PR, but I noticed that the generated filename includes a trailing underscore before the file extension. Maybe update to add the file extension after the `.join("_")`?

**Review by RobertPorterfield — APPROVED** [review]
Great work as always! Very clean. Love how things continuously get broken down into simpler pieces to manage. 10/10 will Approve!

**Pull request merged**